### ISSUE TITLE
Hope to support scheduler cron syntax

### ISSUE THREAD

**Issue by InfernalAzazel** [rq/rq#2080]
1. Can support repetitive tasks
2. Hope to support scheduler cron syntax

**Comment by orokusaki** [comment]
@selwin slightly related to this, quick question if you have a chance:

When running `rqworker --with-scheduler`, after I restart the server (e.g., during deploy), I notice that it takes a while before scheduled tasks are enqueued - [the docs state](https://python-rq.org/docs/scheduling/#running-the-scheduler), "Idle schedulers will periodically (every 15 minutes) check whether the queues they’re responsible for have active schedulers." Is this the delay I'm experiencing? Is there a way to, when starting via `--with-scheduler` tell the system to start running the every-1-second schedule checker immediately?

**Comment by KatHaruto** [comment]
did you try `rq-scheduler` ?

**Comment by InfernalAzazel** [comment]
> did you try `rq-scheduler` ?

I have tried it，If there is me in rq, I don't need to open two command lines。

**Comment by selwin** [comment]
I do plan on adding support for cron like scheduling in RQ 2.x. RQ 2.0 has the necessary infrastructure changes to support this.

**Comment by legout** [comment]
Cron like scheduling + infinite repetition would be awesome. 

One "simple" workaround for cron like scheduling could be, to translate the cron string into several enqueue_at(...., repeat=Repeat(..., interval=xy)), right?

**Comment by selwin** [comment]
Yes, infinite repetition should be easy. We can treat -1 as infinite repetition.

I’m working on cron style repetition, have the basic ready but just need to find the time to polish it up.

**Comment by legout** [comment]
Awesome. Can´t wait for this. ;-)

Using rq and rq-scheduler in parallel works, but its a hassle to run both, especially, because the syntax (e.g. for job/schedule cancelation) is slightly different.

As a sidenote: I am adding rq as an optional task queue to my FlowerPower framework [https://gihub.com/legout/flowerpower](https://gihub.com/legout/flowerpower)
